src/test/scala/com.socrata.tileserver/util/GeoProviderTest.scala

```diff
@@ -31,14 +31,16 @@ class GeoProviderTest extends TestBase with UnusedSugar with MockitoSugar {
 
       val filter = GeoProvider.filter(info.tile, info.geoColumn, Unused)
       val augmented = GeoProvider.augmentParams(info, filter)
+      val augmentedWithQueryTimeout = GeoProvider.addQueryTimeout(augmented, config.TileServerConfig.queryTimeout)
```

Comment:
If you're adding the the query timeout everywhere this should just be part of `augmentParams`.